client/e2e/pages/authenticated/dashboard/profile/page.spec.ts

```diff
@@ -0,0 +1,81 @@
+// 🧪 Suite to test the onboarding\Home page `http://localhost:3000/home`
+
+import { test } from "@playwright/test";
+// 🪄 Page Object Models
+import { DashboardPOM } from "@/e2e/poms/dashboard";
+import { ProfilePOM } from "@/e2e/poms/profile";
+// ☰ Enums
+import { UserRole } from "@/e2e/utils/enums";
+// 🥞 Connection pool to postgres DB
+import { pool } from "@/e2e/utils/pool";
+import { deleteNewUser, upsertIOUser } from "@/e2e/utils/queries";
+// ℹ️ Environment variables
+import * as dotenv from "dotenv";
+dotenv.config({ path: "./e2e/.env.local" });
+
+// 🛠️ Function: deletes the new user record from the database
+const deleteNewUserRecord = async () => {
+  try {
+    const query = deleteNewUser;
+    // Execute the deletion query
+    await pool.query(query);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error("Error deleting new user record:", error);
+    throw error;
+  }
+};
+// 📚 Declare a beforeAll hook that is executed once per worker process before all tests.
+// 🥞 Set DB for profile update
+// For industry_user, ensure there is an associated user
+// For no role/new user, ensure there is NOT an associated user
```

Comment:
I don't follow this comment--for a user there shouldn't be a user?

Reply:
Added more detail: 
`For "new user" test, ensure there is NOT an associated "new user" record in the db so that on "new user" login the ID will have no app_role`